app-ios/tutanota/Sources/Remote/DataWrapper.swift

```diff
@@ -15,6 +15,10 @@ extension DataWrapper: Codable {
     self.data = try decoder.container(keyedBy: Self.CodingKeys).decode(Data.self, forKey: .data)
   }
   
+  public init(from bytes: [UInt8]) {
```

Comment:
this should not be part of the `Codable` extension, you can put it to the next one, which has `wrap()`